lightning/src/ln/peer_handler.rs

```diff
@@ -1266,25 +1304,49 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 	/// [`ChannelManager::process_pending_htlc_forwards`]: crate::ln::channelmanager::ChannelManager::process_pending_htlc_forwards
 	/// [`send_data`]: SocketDescriptor::send_data
 	pub fn process_events(&self) {
+		let mut _single_processor_lock = self.event_processing_lock.try_lock();
```

Comment:
why does the name begin with an underscore if you're actively using the variable?

Reply:
Because the second assignment further down isn't used. rustc complains that this assignment isn't used if you remove the underscore, because its never read from, even though the variable is both written and read to, in total.